packages/table/src/Table.story.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean } from '@storybook/addon-knobs';
+import Table from '.';
+import { Row, Cell, TableHeader, HeaderRow, DataType } from '.';
+import { defaultData, multiRowData } from './storybookdata';
+
+interface DemoDataInterface {
+  name: string;
+  age: number;
+  color: string;
+  location: string;
+}
+
+storiesOf('Table', module)
+  .add('Default', () => (
+    <Table
+      selectable={boolean('isSelectable', true)}
+      data={defaultData}
+      columns={[
+        <TableHeader
+          dataType={DataType.String}
+          sortable
+          label="Name"
+          key="name"
+          stickyColumn
+        />,
+        <TableHeader
+          dataType={DataType.NominalNumber}
+          sortable
+          label="Age"
+          key="age"
+        />,
+        <TableHeader
+          dataType={DataType.String}
+          sortable
+          label="Favorite Color"
+          key="color"
+          accessor="color"
+        />,
+        'Location',
+      ]}
+    >
+      {({ datum }: { datum: DemoDataInterface; rowIndex: number }) => (
```

Comment:
I wonder if we could infer the interface for `datum` by inferring the shape of one object from the `data` prop.

Reply:
Think I got it!